docs/hu2_consultar_ recetas_personalizadas.md

```diff
@@ -0,0 +1,2 @@
+HU2 - Consultar Recetas Personalizadas
```

Comment:
Tenéis que secuenciar las historias de usuario de forma que tengan complejidad creciente.